tests/scene/test_occluder_instance_3d.h

```diff
@@ -0,0 +1,127 @@
+/*  test_occluder_instance_3d.h                                           */
+/**************************************************************************/
+/*                         This file is part of:                          */
+/*                             GODOT ENGINE                               */
+/*                        https://godotengine.org                         */
+/**************************************************************************/
+/* Copyright (c) 2014-present Godot Engine contributors (see AUTHORS.md). */
+/* Copyright (c) 2007-2014 Juan Linietsky, Ariel Manzur.                  */
+/*                                                                        */
+/* Permission is hereby granted, free of charge, to any person obtaining  */
+/* a copy of this software and associated documentation files (the        */
+/* "Software"), to deal in the Software without restriction, including    */
+/* without limitation the rights to use, copy, modify, merge, publish,    */
+/* distribute, sublicense, and/or sell copies of the Software, and to     */
+/* permit persons to whom the Software is furnished to do so, subject to  */
+/* the following conditions:                                              */
+/*                                                                        */
+/* The above copyright notice and this permission notice shall be         */
+/* included in all copies or substantial portions of the Software.        */
+/*                                                                        */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,        */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF     */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. */
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY   */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,   */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE      */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
+/**************************************************************************/
+
+#ifndef TEST_OCCLUDER_INSTANCE_3D_H
+#define TEST_OCCLUDER_INSTANCE_3D_H
+
+#include "core/math/vector3.h"
+#include "scene/3d/mesh_instance_3d.h"
+#include "scene/3d/occluder_instance_3d.h"
+#include "scene/main/scene_tree.h"
+#include "scene/resources/3d/primitive_meshes.h"
+
+#include "tests/test_macros.h"
+
+namespace TestOccluderInstance3D {
+
+TEST_CASE("[SceneTree][OccluderInstance3D] Test baking functionality") {
+// We need to load an occluder path because when the light occluder generates
+// it has to save to a file.
+#ifdef WINDOWS_ENABLED
+	const String occluder_path = OS::get_singleton()->get_environment("TEMP").path_join("test_occluder.occ");
+#else
+	const String occluder_path = "/tmp/test_occluder.occ";
+#endif
```

Comment:
As of https://github.com/godotengine/godot/pull/98397, this can be replaced with a call to `OS::get_temp_dir()` on all platforms:

```suggestion
// We need to load an occluder path because when the occluder generates,
	// it has to save to a file.
	const String occluder_path = OS::get_singleton()->get_temp_path().path_join("test_occluder.occ");
```